lib/adb.js

```diff
@@ -50,6 +50,28 @@ class ADB {
     }
     this.executable.defaultArgs.push('-P', this.adbPort);
   }
+
+  /**
+   * Create a new instance of `ADB` that inherits configuration from this `ADB` instance.
+   * This avoids the need to call `ADB.createADB()` multiple times.
+   * @param {object} opts - Additional options mapping to pass to the `ADB` constructor.
+   * @returns {ADB} The resulting class instance.
+   */
+  clone (opts = {}) {
+    const cloneOptions = _.defaultsDeep(opts, _.cloneDeep(this));
+
+    // Reset default arguments created in the constructor
```

Comment:
why this is necessary?

Reply:
Without this code, `-H` and `-P` can be injected into `defaultArgs` multiple times.

Reply:
I think we might just add this statement as a comment to make it more obvious